debian/rules

```diff
@@ -57,10 +57,10 @@ override_dh_makeshlibs:
 # Create dbg packages.
 # FIXME: should we just have a rdma-core-dbg package ?
 override_dh_strip:
-	dh_strip -plibibcm1 --dbg-package=libibcm1-dbg
-	dh_strip -plibibumad3 --dbg-package=libibumad3-dbg
-	dh_strip -plibibverbs1 --dbg-package=libibverbs1-dbg
-	dh_strip -plibrdmacm1 --dbg-package=librdmacm1-dbg
+	dh_strip -plibibcm1 --dbgsym-migration='libibcm1-dbg (<< 12-1~)'
+	dh_strip -plibibumad3 --dbgsym-migration='libibumad3-dbg (<< 12-1~)'
+	dh_strip -plibibverbs1 --dbgsym-migration='libibverbs1-dbg (<< 12-1~)'
+	dh_strip -plibrdmacm1 --dbgsym-migration='librdmacm1-dbg (<< 12-1~)'
 
```

Comment:
This seems like a good idea, but this breaks building on Jessie:

```
dh_strip -plibibcm1 --dbgsym-migration='libibcm1-dbg (<< 12-1~)'
dh_strip: unknown option; aborting
debian/rules:60: recipe for target 'override_dh_strip' failed
make[1]: Leaving directory '/home/jgg/src'
make[1]: *** [override_dh_strip] Error 25
debian/rules:4: recipe for target 'binary' failed
```

Which means at a minimum this patch should update buildlib/cbuild to remove jessie and include some other acceptable Debian distribution for testing..